src/main/java/org/javawebstack/orm/mapper/DefaultMapper.java

```diff
@@ -81,12 +109,14 @@ public SQLType getType(Class<?> type, int size) {
     public String getTypeParameters(Class<?> type, int size) {
         if (type.isEnum())
             return Arrays.stream(((Class<? extends Enum<?>>) type).getEnumConstants()).map(c -> "'" + c.name() + "'").collect(Collectors.joining(","));
-        if (type.equals(String.class))
-            return size > 255 || size < 1 ? null : String.valueOf(size);
+        if (type.equals(String.class) || type.equals(char[].class))
+            return size > 65535 || size < 1 ? null : String.valueOf(size);
```

Comment:
Return MEDIUMTEXT (65535+) or LONGTEXT (16777216+)

Reply:
Thanks! Your totally right! This section is for the parameters so null for this area good as these Text types are not restrictable either way. You can see the changes for the types here though: https://github.com/JavaWebStack/ORM/pull/6/commits/028b38c4fb5db45cd699921a31f14d668e7b2968